README.md

```diff
@@ -29,8 +29,14 @@ Want to take your Watson app to the next level? Looking to leverage Watson Brand
 
 # Watch the Video
 
+#### Configure and run Watson Conversation Slots Intro
```

Comment:
how about "Running this application with Cloud Foundry on Bluemix"

Reply:
video does show "Deploy to Bluemix", but also local deploy. Does that matter?

Reply:
in the end it runs as an application with CF on Bluemix, so i don't think it's misleading :)